python/ray/actor.py

```diff
@@ -222,12 +221,15 @@ def actor_method_executor(dummy_return_id, actor, *args):
     return actor_method_executor
 
 
-def fetch_and_register_actor(actor_class_key, worker):
+def fetch_and_register_actor(actor_class_key, resources, worker):
     """Import an actor.
 
     This will be called by the worker's import thread when the worker receives
     the actor_class export, assuming that the worker is an actor for that
     class.
+
+    Args:
+    TODO!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
nice

Reply:
This is kind of silly, but when I need to remember to do something I introduce linting errors so that Travis will remind me.